app/models/game.rb

```diff
@@ -0,0 +1,47 @@
+class Game < ActiveRecord::Base
+  has_many :pieces
+  has_many :players
+
+  after_create :populate_board!
+
+  def populate_board!
+
+
+    # Black Pieces of the board
+    (0..7).each do |i|
+      Pawn.create(game_id: id, x_position: i, y_position: 1, color: "black")
+    end
+
+    Rook.create(game_id: id, x_position: 0, y_position: 0, color: "black")
```

Comment:
@lasley I meant over here - for each Peace.create in the params, shouldn't it be `game_id: :id` ?